samza-test/src/test/scala/org/apache/samza/test/integration/StreamTaskTestUtil.scala

```diff
@@ -122,12 +104,36 @@ object StreamTaskTestUtil {
     TestTask.reset()
   }
 
+  var servers: Buffer[KafkaServer] = null
+
   def beforeSetupServers {
-    zookeeper = new EmbeddedZookeeper(zkConnect)
-    server1 = TestUtils.createServer(new KafkaConfig(props1))
-    server2 = TestUtils.createServer(new KafkaConfig(props2))
-    server3 = TestUtils.createServer(new KafkaConfig(props3))
-    zkClient = new ZkClient(zkConnect + "/", 6000, 6000, ZKStringSerializer)
+    zookeeper = new EmbeddedZookeeper()
+    zkUtils = ZkUtils(zkConnect, zkSessionTimeout, zkConnectionTimeout, JaasUtils.isZkSecurityEnabled())
+
+    val props = TestUtils.createBrokerConfigs(3, zkConnect, true)
```

Comment:
One thing that I am curious about the broker config, did you turn on the new message format w/ timestamp support at the broker side in this patch?